### PR TITLE
Feature/plan oic 56394 show server error

### PR DESCRIPTION
**Pull request by paullanoracle**

### CODE REVIEW

**Review by tacobacca — APPROVED** [review]

File: src/logger.ts [hunk 1]
```diff
@@ -75,6 +79,14 @@ class Logger {
     return JSON.stringify(val, null, 2);
   }
 
+  formatV2(val: any): string {
+    try {
```
Comment (sebastien-ma) [inline src/logger.ts:83]: Just put this in format?

File: src/logger.ts [hunk 2]
```diff
@@ -59,6 +59,10 @@ class Logger {
     return this.log('info', msg);
   }
 
+  infoServer(data: any): Logger {
```
Comment (sebastien-ma) [inline src/logger.ts:62]: It is not necessary to have this method in logger class. If you want to print for specific pattern like server response, please have the common method in api.ts

**Review by sebastien-ma — CHANGES_REQUESTED** [review]

File: src/logger.ts (hunk 2 above)
Comment (paullanoracle) [reply src/logger.ts:62]: Will change in the next commit.

File: src/logger.ts (hunk 1 above)
Comment (paullanoracle) [reply src/logger.ts:83]: Will change in the next commit.

**Pull request merged**